### PR TITLE
Started a jobboard

### PR DESCRIPTION
**Pull request by jdamerow**
What do you think? We would need a Google form and either a job to removed expired jobs or just do it by and every once in a while.

### CODE REVIEW

**Comment by ColeDCrawford** [comment]
Would we be pulling these manually from the form? Otherwise we could write up a script to do it on a cron job basis with Actions.

**Comment by robcast** [comment]
Looks good to me. Does the list sort automatically by date, or is this done by the person editing the page?

**Comment by jdamerow** [comment]
It should put them in the order they are in in the yml file. We can probably sort it by date fairly easily but that's as far as I got.

**Comment by ColeDCrawford** [comment]
I think the link to the form is missing? Other than that, this looks good to me!

I'd be interested in automating this - a script to get new jobs from the Google Form, and then open a PR if they have changed or expired. We can also have a status column not on the Form but in the Sheet in case we want to reject any. But that can come a bit later.

File: content/job-board.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+title = "Job Board"
+type="jobboard"
++++
+
+## Jobs in the Digitial Humanities Tech Space
+
+Do you have a job to submit to the board? Please fill out this form.
```
Comment (ColeDCrawford) [inline content/job-board.md:8]: Missing link?

**Review by ColeDCrawford — APPROVED** [review]

File: content/job-board.md (hunk 1 above)
Comment (jdamerow) [reply content/job-board.md:8]: Ya, I didn't get around to making the form yet.

Comment (jdamerow) [reply content/job-board.md:8]: Hence the draft status of the PR 😆

**Comment by maltevogl** [comment]
lgtm on render. Good to go I would say..

**Review by maltevogl — APPROVED** [review]
the form links work, the banner looks nice, and there is a first job . good to go.

**Pull request merged**